### ISSUE TITLE
`pulumi install` doesn't work in a policy pack directory.

### ISSUE THREAD

**Issue by julienp** [pulumi/pulumi#16437]
One thing I missed in the review that I just tried: `pulumi install` doesn't work in a policy pack directory.

```
$ mkdir policypack && cd policypack
$ pulumi policy new aws-typescript --generate-only
$ pulumi install
error: no Pulumi.yaml project file found (searching upwards from /Users/justin/Documents/2024/06/20/policypack). If you have not created a project yet, use `pulumi new` to do so: no project file found
```

I guess our options are:
1. Leave the lang-specific list of commands for `pulumi policy new` (not ideal)
2. Update `pulumi install` to work when there is a `PulumiPolicy.yaml` file rather than `Pulumi.yaml`
3. Create a new command (e.g. `pulumi policy install`) that is basically the same as `pulumi install` but just for policies.

I think I prefer (2) over (3), but haven't looked into the details of the implementation. The one difference is that I don't think we need to do pulumi plugin installs for policy packs. We just need to install the language's dependencies.

_Originally posted by @justinvp in https://github.com/pulumi/pulumi/pull/16411#discussion_r1647590897_

**Comment by mikhailshilkov** [comment]
This issue has been shipped in [release v3.121.0](https://github.com/pulumi/pulumi/releases/tag/v3.121.0).

**Comment by mikhailshilkov** [comment]
This issue has been addressed in PR [#16438](https://github.com/pulumi/pulumi/pull/16438) and shipped in [release v3.121.0](https://github.com/pulumi/pulumi/releases/tag/v3.121.0).
